lib/decoration.rb

```diff
@@ -0,0 +1,7 @@
+module Decoration
```

Comment:
Did I mention that you probably don't need this anymore, draper should decorate active record results.